django_socio_grpc/tests/test_async_basic_service.py

```diff
@@ -45,3 +45,9 @@ async def test_basic_service_attr_name_on_inheritance(self):
         request.results.append(alice)
         response = await grpc_stub.BasicList(request)
         self.assertEqual(len(response.results), 2)
+
+    async def test_no_meta_serializer(self):
```

Comment:
now that you have cleaned the code do the unit test doen't need to move from serializer message to testing _MessageToData subclass ?